File: app/src/main/java/com/hoymm/root/tictactoe2/GameEngine/GameFragsCommunication.java
package com.hoymm.root.tictactoe2.GameEngine;

import com.hoymm.root.tictactoe2.R;

/**
 * Created by hoymm on 10.11.17.
 */

public interface GameFragsCommunication {
    String getWhosTurnNow();
    boolean isCircleTurnNow();
    boolean isYourTurnNow();
}

File: app/src/main/java/com/hoymm/root/tictactoe2/GameEngine/GameFragsCommunication.kt
package com.hoymm.root.tictactoe2.GameEngine

/**
 * Created by hoymm on 10.11.17.
 */

interface GameFragsCommunication {
    val whosTurnNow: String
    val isCircleTurnNow: Boolean
    val isYourTurnNow: Boolean
}
